.travis.yml

```diff
@@ -0,0 +1,209 @@
+os: linux
+dist: xenial
+language: java
+jdk:
+  - openjdk11
+  
+addons:
+  firefox: "76.0.1"
+  
+services:
+  - xvfb
+  - docker
+
+  
+cache:
+  directories:
+    - $HOME/.m2
+# the cache can grow constantly
+before_cache:
+  - rm -rf $HOME/.m2/repository/org/alfresco/alfresco-share*
+  
+branches:
+  only:
+    - master
+    - /support\/.*/
+    - fix/Travis-share-build
+    
+stages:
+  - build
+  - tests
+    if: commit_message !~ /\[skip-tests\]/
+  - release
```

Comment:
In Bamboo we have the possibility to choose between "Engineering Release" (that pushes the distribution zip to Nexus) and "Company Release" (that pushes the Docker image to DockerHub).